src/purchases.ts

```diff
@@ -537,6 +538,17 @@ export default class Purchases {
         }
     }
 
+    /**
+     * Enable automatic collection of Apple Search Ad attribution. Disabled by default
+     * @returns {Promise<void>} The promise will be rejected if setup has not been called yet.
+     */
+    public static async enableAdServicesAttributionTokenCollection(): Promise<void> {
+        if (Platform.OS === "ios") {
```

Comment:
Should we log something if it's not? Also, we could add that this is iOS only to the documentation above (even if it's pretty self-explanatory mentioning Apple Search Ads).

Reply:
Added to the doc! And I also thought about logging something but we also don't log anything for the `setAutomaticAppleSearchAdsAttributionCollection` right now 🤷‍♂️  

Also, do we have a logging method to in RN? I couldn't find an existing utility 🤔

Reply:
No idea... Haven't touch RN much yet. But if that's how we have it right now for other methods, I think it's ok for now.